### ISSUE TITLE
Macrocosm Logo

### ISSUE THREAD

**Issue by DylanMoriarty** [developmentseed/macrocosm#25]
In following the last few discussions on the logo, here are a couple more proposed logos for Macrocosm. 

<img width="576" alt="screenshot 2016-07-28 17 23 17" src="https://cloud.githubusercontent.com/assets/9341928/17229989/0dc20e34-54e8-11e6-92af-0a63e4bb7380.png">

This design solves a few of the issues from the last round, both in giving more connections, and doesn't create a misleading random bowtie shape. It's still missing the 'universal' element, but that maybe that's ok.

---

For reference, here's some older working versions that led us to this design.

<img width="433" alt="screenshot 2016-07-28 17 23 36" src="https://cloud.githubusercontent.com/assets/9341928/17229990/0dc4aa9a-54e8-11e6-90ba-916ccdf666b0.png">

@dereklieu

**Comment by felskia** [comment]
I like the top versions and the one on the left hand side more. The only think I would say is that I would make the M smaller so that it works better with the text below. 

<img width="239" alt="screen shot 2016-07-29 at 4 39 41 pm" src="https://cloud.githubusercontent.com/assets/5421566/17262752/53be9f9c-55ab-11e6-9b43-2e35a4f372a6.png">

**Comment by dereklieu** [comment]
Yup, good work @DylanMoriarty thanks for the effort here. I agree with Ali on the slightly smaller size. Can you post versions with and without the text?

**Comment by DylanMoriarty** [comment]
@dereklieu: Here is the logo with the 'M' smaller, and a zip file containing the logo & variants. 

<img width="579" alt="screenshot 2016-08-05 11 34 38" src="https://cloud.githubusercontent.com/assets/9341928/17441880/1e1cd8a8-5b01-11e6-8ed3-bfdb3fc60ec1.png">


[macrocosm.zip](https://github.com/developmentseed/macrocosm/files/404073/macrocosm.zip)

**Comment by dereklieu** [comment]
Cool thanks!

**Comment by DylanMoriarty** [comment]
@dereklieu: Here is the logo with the 'M' smaller, and a zip file containing the logo & variants. 

<img width="263" alt="screenshot 2016-08-05 12 04 50" src="https://cloud.githubusercontent.com/assets/9341928/17442713/071361dc-5b05-11e6-96b4-a07e92fb8680.png">

[macrocosm.zip](https://github.com/developmentseed/macrocosm/files/404144/macrocosm.zip)
